stable/xray/ci/global-section.yaml

```diff
@@ -0,0 +1,49 @@
+unifiedUpgradeAllowed: true
```

Comment:
@rahulsadanandan  can you append the file name `global-section-values.yaml`

Reply:
Changed